ionoscloud/resource_ipfailover.go

```diff
@@ -28,6 +34,7 @@ func resourceLanIPFailover() *schema.Resource {
 			"ip": {
 				Type:             schema.TypeString,
 				Required:         true,
+				ForceNew:         true,
```

Comment:
we should test manually an upgrade, to make sure this doesn't break anything

Reply:
@cristiGuranIonos I've done this and it works properly.